llms/mlx_lm/models/stablelm.py

```diff
@@ -128,7 +128,6 @@ class DecoderLayer(nn.Module):
     def __init__(self, config: ModelArgs):
         super().__init__()
         self.self_attn = Attention(config=config)
-        self.input_layernorm = LayerNorm(config.hidden_size, eps=config.layer_norm_eps)
```

Comment:
@awni this is duplicate right?

Reply:
Yea, good catch! Would be great if you remove!